apisix/plugins/google-cloud-logging.lua

```diff
@@ -61,13 +61,13 @@ local schema = {
                     type = "string",
                     default = "https://logging.googleapis.com/v2/entries:write"
                 },
-                ssl_verify = {
```

Comment:
I don't know whether this plugin is used by someone, if we just remove this setting, it's bad for these users. Or we can just reserve and deprecate it.

Reply:
I think this has no substantive significance. In most cases, this field remains the default setting, and general users will not actively set this option. this plugin is a new feature released in version 2.11. In such a short period of time, I don't think it will be used in a mass production environment. In addition, it is a bug for people who use file configuration ^_^

Reply:
Got it.